src/scrape_up/icc/icc_rankings.py

```diff
@@ -93,3 +95,83 @@ def player_ranking(self, type, format):
             return response_list
         except:
             return None
+
+    def team_rankings_women(self, format):
+        """
+        Create an instance of `ICC` class.\n
+        Required Params - `format` - "ODI","T20"
+        ```python
+        icc = ICC()
+        icc.team_rankings_women(format="odi")
+        ```
+        ```js
+        [
+            {
+                "rank":1,
+                "team":"Australia"
+            }
+        ]
+        ```
+        """
+        try:
+            format = format.lower()
+            obj_keys = ["rank", "team"]
+            resposne_list = []
+            url = self.url_women + "team-rankings/" + format
+            response = requests.get(url)
+            soup = BeautifulSoup(response.content, "html.parser")
+            teams = soup.find_all("span", class_="u-hide-phablet")
+            for rank, team in enumerate(teams, 1):
+                obj_values = [rank, team.get_text()]
+                resposne_list.append(dict(zip(obj_keys, obj_values)))
+
+            return resposne_list
+        except:
+            return None
+
+    def player_ranking_women(self, type, format):
+        """
+        Create an instance of `ICC` class.\n
+        Required Params
+        - `format` - "ODI","T20"
+        - `type` - "batting","bowling" or "all-rounder"\n
+        ```python
+        icc = ICC()
+        icc.player_ranking_women(format="test",type="batting")
+        ```
+        Returns \n
+        ```js
+        [
+            {
+                "rank":1,
+                "team":"Kane Williamson"
+            }
```

Comment:
The method isn't working properly, also the docstring is not updated with the response.  I do not think Kane Williamson belongs to women playing nation.

![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/7e5bc646-a818-42cf-a629-7e0d1cab2624)

Reply:
![image](https://github.com/Clueless-Community/scrape-up/assets/89679280/ddd2150e-f5ef-4df9-917f-b147549d7173)
The method is now working and the updated docstring is as this:
[
            {
                "rank":1,
                "team":"Natalie Sciver-Brunt"
            }
            ...
]